editions/tw5.com/tiddlers/webserver/Using the external JavaScript template.tid

```diff
@@ -1,23 +1,128 @@
 created: 20180905075846391
-modified: 20200814110131083
+modified: 20210611055708739
 tags: [[WebServer Guides]]
 title: Using the external JavaScript template
 type: text/vnd.tiddlywiki
 
-Setting the [[root-tiddler|WebServer Parameter: root-tiddler]] parameter to `$:/core/save/all-external-js` switches to a special template that externalises TiddlyWiki's core ~JavaScript into a separate file. For example:
+\define break-section()
```

Comment:
It's really important that the TiddlyWiki documentation is consistent. Here you've introduced a new system for navigating within this tiddler. That may be a useful innovation, but only if it is used consistently; allowing each documentation tiddler to evolve their own non-standard navigation techniques makes things difficult for users.

So, here, either we should use one long tiddler, or we should break it up into separate tiddlers.